Generator/generator-botbuilder-java/generators/app/templates/core/src/main/java/package-info.java

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I didn't see the other 2 having the package-info. Is this right?

Reply:
That's a good question. I am not sure, I copied it just in case.